pl_gugik/pl_gugik_README.txt

```diff
@@ -0,0 +1,21 @@
+# pl_gugik_README.txt
+
+The files in this section result from the conversion of datasets originating
+from [GUGiK](http://www.gugik.gov.pl)
+
+## Included grids
+
+### Poland vertical grid:
+
+*Source*: [GUGiK](http://www.gugik.gov.pl/bip/prawo/modele-danych)
+*Format*: GeoTIFF converted from 'XYZ ASCII Grid'
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c) Główny Urząd Geodezji i Kartografii
```

Comment:
I'm just adding 2 spaces at the end of those lines, so they appear nicely in github
```suggestion
*Source*: [GUGiK](http://www.gugik.gov.pl/bip/prawo/modele-danych)  
*Format*: GeoTIFF converted from 'XYZ ASCII Grid'  
*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
*Credit*: (c) Główny Urząd Geodezji i Kartografii  
```